### DISCUSSION TITLE
烧纸钱 | /posts/paper-money/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#587] [Announcements]
https://yuanfan.rbind.io/posts/paper-money/

**Comment by yihui** [comment]
是条汉子！头一回听说还有人为了好玩而买纸钱！

**Reply by earfanfan** [reply]
唉，你这夸的，莫名其妙地让我瞬间觉得自己一点也不酷，还有点傻愣愣的。

画蛇添足地讲，就像是，有一天早上我骑车去上班，由于刚下过雨地面湿滑，骑到一个地方时明显感觉自行车龙头不稳，我挣扎了几下就跳车了，几乎同时车也摔在地上，刚好对面有个大叔坐在他的电动车上看到了全程，他哈哈笑个不停，一边笑还一边指着我。一般来讲，受传统思想束缚的国人都会收敛情绪，然而那个大叔笑了好多秒，还笑得很开怀，就是真得觉得好笑……

**Reply by CyrusYip** [reply]
@yihui 明明是认真、好奇、有趣的大人。

> 其实今天清早出门时也看见了，当时想买可能只是因为好奇，心里有一小丢丢对这传统信仰的不屑。傍晚还想买，那当然就是凑过去问啦。

这是好奇。

> 不过当我经由摊主们还有我妈的言语见识到了她们对鬼神的敬畏后，决定也尊重一下那些纸钱，不要浪费，回头认真在地上画个带门的圈圈，然后在圈里烧纸钱。

这是认真。

> 那么尊重纸钱们，把它们拿去认真烧掉也是一种人生体验啊。所以就当是烧给狗和狗主的吧。

这是有趣。看到这我快要笑出来，纸钱是烧给逝者的，怎么会有人烧给生者。不过只要不信这一套的话，烧给自己也无所谓了。

**Reply by earfanfan** [reply]
我觉得我身上存在某种鲜明的中国人特性，一般呢，受夸奖是乐意的，但是像益辉这样夸得我觉得奇怪的，就一股脑再怼回去，而像叶寻同学这样夸得我觉得太好的，又感到有点不好意思了，哈哈哈哈

**Reply by yihui** [reply]
我写的时候并没有想着要夸的意思，只是说狗主实在不同寻常。一般来说中国人应该都是很忌讳与死有关的东西的，但狗主却似乎与此类忌讳绝缘，而是纯粹用赤子之心去看待这些东西。若要问我敢不敢买纸钱玩，我可能真的会犹豫一下，因为这些东西对我来说还是有点童年阴影。

**Reply by CyrusYip** [reply]
> 一般来说中国人应该都是很忌讳与死有关的东西的

@yihui 生老病死是每个人都要经历的阶段。我觉得有些忌讳过头了，似乎有很深的死亡恐惧。一两个月前我发现选车牌号有个排除含4的选项，没想到这种忌讳到这个程度了。以前我还看过楼梯没有4楼、有人把家门4楼的牌子弄掉。我觉得没这些忌讳人会活得比较轻松。

没这样忌讳也能让了解死亡成为可能。我小时候对于亲人死亡是很迷茫的，就像一个人突然不见了，然后变成骨灰。

说到死亡的忌讳，我还在农村老家发现办葬礼的家庭和旁人之间的矛盾。假如有人去世，最高级的葬礼大概是游街哭丧、奏乐、烧鞭炮、撒纸钱。对那个家庭来说，这算是很孝顺体面的做法。有旁人认为这样张扬的葬礼不吉利（我不知道这是不是普遍观点）。纸钱和鞭炮碎屑飘到自家门口赶紧扫走，经过那家人时会绕道，丧乐听久了觉得不舒服。大家都觉得葬礼很重要，但又有人会讨厌别人的葬礼。我也不喜欢张扬的葬礼，倒不是因为忌讳，是因为太吵了扰民，有时候我睡着觉就被吵醒。

**Reply by earfanfan** [reply]
>若要问我敢不敢买纸钱玩，我可能真的会犹豫一下，因为这些东西对我来说还是有点童年阴影。

我有点好奇是撒事给益辉留下了童年阴影……不过我记得以前在哪篇博客下面，叶寻同学提过白天看恐怖片也会感到害怕，不造现在咋样？我记得也是很久以前，列弛提到鬼怪之事，我当时让他发给我瞅瞅，看完以后也有一小丢丢觉得可怕。尽管如此，还是好奇。

**Reply by CyrusYip** [reply]
@earfanfan 现在也怕恐怖片。之前鼓起勇气和女朋友看了《孤儿怨》和《恐怖游轮》，剧情很精彩，看的时候很怕，看完几天还有些残留的恐惧。

**Reply by earfanfan** [reply]
@CyrusYip
我估计益辉的童年阴影应该是不咋恐怖的，跟《恐怖游轮》比肯定不是一个量级，你连恐怖电影都敢看，哈哈哈，那就等着益辉给咱们讲讲阴影是撒吧。

**Reply by yihui** [reply]
阴影不算太恐怖，就是小时候见过几次村里死人（但并没见过死者本人，至今我都没亲眼见过任何死者），葬礼让人感觉很阴森，加上大人讲些鬼怪的事情，比如死了的人夜里还会回来摸我们，这就会让我对花圈、坟墓、棺材（甚至黑色的木头）之类的东西有种本能的恐惧。当然，后来长大了肯定就不像小时候那样怕了。如果现在让我去买纸钱回来玩，我是可以买了玩的，只是我自己肯定不会主动想起来买了玩。

**Reply by earfanfan** [reply]
@yihui
>如果现在让我去买纸钱回来玩，我是可以买了玩的，只是我自己肯定不会主动想起来买了玩。

哈哈，没有这个如果。我玩我的，又不会逼你跟我一样……

**Reply by yihui** [reply]
如果下次点一首没有如果，请问就可以有如果了吗？

**Reply by earfanfan** [reply]
@yihui

哈，快把你的年更遥提上日程，俺们要看。不然的话，你是风，你是火，你是织网滴恶魔……

**Reply by yihui** [reply]
本人已死，有事烧纸。

**Reply by earfanfan** [reply]
@yihui

嘿，没事，我抽空去把你捞上来，该耕还是要耕的说。
